src/sidebar/components/ShareDialog/test/ShareDialog-test.js

```diff
@@ -59,14 +60,11 @@ describe('ShareDialog', () => {
   });
 
   function selectTab(wrapper, name) {
-    wrapper
-      .find(`Tab[aria-controls="${name}-panel"]`)
-      .find('button')
-      .simulate('click');
+    wrapper.find(`button[aria-controls="${name}-panel"]`).simulate('click');
   }
 
   function getActiveTab(wrapper) {
-    return wrapper.find('Tab').filter({ selected: true });
+    return wrapper.find(Tab).filter({ selected: true });
```

Comment:
These changes are not directly related with the other changes in this PR, but this test started to fail after frontend-shared was updated.

My theory is that [the changes](https://github.com/hypothesis/frontend-shared/pull/1238/files) we did in how we export components, are causing the names of exported components to no longer work as before, so `'Tab'` is no longer found by enzyme.

As a workaround we can use the component itself, but we need to investigate this as it will probably affect other test in other projects.

Reply:
More context: It is related with the change I mentioned, but it affects `Tab` specifically because there's other `Tab` component in client.

Rollup is renaming the one from frontend-shared to `Tab$1` to avoid conflicts in the final bundle.